### DISCUSSION TITLE
QUIC API changes in 0.23.0 release

### DISCUSSION THREAD

**Discussion by msk** [aicers/giganto#911] [Q&A] [answered]
We're using Giganto in our application at @petabi, and we're trying to update our client to be compatible with the latest 0.23.0 release. However, we're having trouble finding information on what changes were made to the QUIC API.

The [CHANGELOG](https://github.com/aicers/giganto/blob/0.23.0/CHANGELOG.md#changed) mentions updates to "the compatibility version of the quic communication modules", but it doesn't provide any details on what specifically changed in the QUIC API. The GraphQL API changes are well-documented, but we're primarily using the QUIC API in our application.

Could someone please provide more information on what changes were made to the QUIC API in the 0.23.0 release? What specific changes do we need to make to our client to ensure compatibility with this new version?

**Comment by sophie-cluml** [comment, marked as answer]
Hello, msk.

The version of giganto-client that Giganto depends on has been changed from 0.20.0 to 0.21.0.

Thank you for pointing this out and I apologize for any confusion caused  and will create an issue to address this improvement promptly to update the CHANGELOG to mention this change.

**Reply by msk** [reply]
Then I assume that updating our giganto-client dependency to 0.21.0, which @syncpark has already completed, is all we need to do to be compatible with Giganto 0.23.0. Thanks for clarifying the change, and I appreciate your willingness to update the CHANGELOG to include this information for future reference!

**Comment by kimhanbeom** [comment]
This discussion will be closed as the question has been answered.
